### PR TITLE
or-tools 9.1

### PR DESCRIPTION
**Pull request by theirix**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by Moisan** [comment]
Upstream changelog states:
>*Dependencies Update*
>Protobuf 3.15.8 -> 3.18.0.

I would wait until we merge https://github.com/Homebrew/homebrew-core/pull/85282 before merging this PR.

**Review by carlocab — APPROVED** [review]
#85282 looks like it'll need a rebase anyway, and the tests pass here, so I don't think we need to delay this, @moisan.

**Pull request closed without merging**
